.github/workflows/acc-dockeronly-nightly.yml

```diff
@@ -0,0 +1,30 @@
+name: Acceptance Tests Docker Only
+
+on:
+  # Change to nightly cadence once API-credential-requiring tests are added to the jobs
+  workflow_dispatch: 
+
+jobs:
+  plugins-database:
```

Comment:
This is a nice pattern. I like how we're getting package level test isolation and parallelization. That'll make re-running easier in the future when trying to diagnose issues.